src/Common/src/Interop/Unix/System.Native/Interop.ForkAndExecProcess.cs

```diff
@@ -15,9 +15,10 @@ internal static partial class Sys
         internal static unsafe void ForkAndExecProcess(
             string filename, string[] argv, string[] envp, string cwd,
             bool redirectStdin, bool redirectStdout, bool redirectStderr,
-            out int lpChildPid, out int stdinFd, out int stdoutFd, out int stderrFd)
+            out int lpChildPid, out int stdinFd, out int stdoutFd, out int stderrFd, out int res, bool shouldThrow = true)
```

Comment:
>out int res [](start = 85, length = 11)

we can make this method just return the result instead of passing it as out param